huaweicloud/services/rds/resource_huaweicloud_rds_instance.go

```diff
@@ -937,8 +937,8 @@ func GetRdsInstanceByID(client *golangsdk.ServiceClient, instanceID string) (*in
 
 func buildRdsInstanceAvailabilityZone(d *schema.ResourceData) string {
 	azList := make([]string, len(d.Get("availability_zone").([]interface{})))
```

Comment:
It is easier to assign the result directly to azList.

```suggestion
	azList := utils.ExpandToStringList(d.Get("availability_zone").([]interface{}))
```

Reply:
done